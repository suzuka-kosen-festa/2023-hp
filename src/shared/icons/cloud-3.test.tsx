import { composeStory } from "@storybook/react";
import { render } from "@testing-library/react";
import Meta, { Default } from "./cloud-3.stories";

describe("[shared/icons] cloud-3.tsx", () => {
  it("should render the Cloud component", () => {
    const Composed = composeStory(Default, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
});
