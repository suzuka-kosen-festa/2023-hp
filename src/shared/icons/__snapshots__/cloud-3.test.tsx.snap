// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[shared/icons] cloud-3.tsx > should render the Cloud component 1`] = `
<div>
  <svg
    fill="none"
    viewBox="0 0 434 228"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M157.643.36c-22.581.838-23.615 1.04-26.749 5.231-3.531 4.72-3.715 5.239-3.715 10.463 0 6.789.422 7.933 4.662 12.62l4.284 4.738 72.676.007 72.675.009 4.677 5.29c6.371 7.208 6.451 16.893.194 23.31-3.887 3.985-5.984 4.624-15.732 4.793-23.136.4-26.163 1.564-30.27 11.642-2.043 5.01-1.982 6.5.484 11.83 1.553 3.353 4.257 6.886 6.011 7.847 4.42 2.426 172.601 2.673 178.452.262 5.17-2.13 9.927-13.168 8.427-19.55-.561-2.388-3.558-6.094-6.66-8.237-5.322-3.677-7.357-3.895-36.02-3.855-18.467.026-31.718-.693-33.793-1.83-8.524-4.678-9.764-18.58-2.36-26.502 3.971-4.252 5.922-4.939 14.072-4.96 12.908-.031 16.055-1.154 19.37-6.909 3.852-6.684 3.508-15.653-.801-20.968l-3.536-4.36L282.561.36c-55.787-.48-112-.48-124.918 0ZM39.968 127.679c-16.838.49-31.85 1.514-33.36 2.276C3.087 131.73 0 138.684 0 144.838c0 3.153 1.901 6.665 5.527 10.207l5.528 5.401 30.163.066c33.078.071 37.905 1.179 41.84 9.6 2.86 6.115 1.397 15.262-3.109 19.442-2.178 2.021-7.643 3.515-15.883 4.342-14.863 1.493-16.042 2.105-19.253 9.983-1.913 4.695-2.05 7.319-.614 11.783 3.985 12.378-.79 11.754 90.386 11.806l82.053.046 4.782-5.487c6.315-7.247 6.257-14.144-.178-21.534l-4.96-5.696-32.702-.871-32.702-.872-3.429-4.714c-3.99-5.484-5.22-13.328-2.597-16.568 1.007-1.244 1.242-2.26.522-2.26-.718 0 .759-1.962 3.281-4.36 3.49-3.317 6.502-4.384 12.58-4.462 4.395-.056 46.278-.075 93.076-.044 81.314.054 85.326-.092 90.52-3.332 6.512-4.06 8.354-10.2 5.741-19.141-1.348-4.612-3.377-7.223-6.718-8.642-5.1-2.166-243.259-3.617-303.886-1.852Z"
      fill="#9B84A3"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
